db-client-java/src/main/java/com/eventstore/dbclient/ExpectedRevision.java

```diff
@@ -5,17 +5,51 @@
 
 import java.util.Objects;
 
+/**
+ * Constants used for expected revision control.
+ * <p>
+ * The use of expected revision can be a bit tricky especially when discussing guaranties given by the
```

Comment:
**SUGGESTION:** I think that this line can be safely removed, as it's more statement than specific information 😉 
```suggestion

```